test/page_live_test.exs

```diff
@@ -16,7 +16,7 @@ defmodule AshAdmin.Test.PageLiveTest do
     assert html =~ "body"
     assert html =~ "String"
 
-    {:ok, _view, html} = live(conn, "/api/admin/test")
+    {:ok, _view, html} = live(conn, "/api/admin?&resource=test")
 
```

Comment:
why did the routes change? Also, this should probably be `?resource=test` not `?&resource=test` right?

Reply:
sorry, it is a typo, fixed